client/src/components/Recipe/RecipeIngredients.js

```diff
@@ -9,20 +10,23 @@ const Ingredient = styled.ul`
   }
 `;
 
-const RecipeIngredients = () => {
+const RecipeIngredients = ({ Ingredients, id }) => {
+  console.log(Ingredients);
   return (
     <>
       <h2>Zutaten</h2>
       <Ingredient>
-        <li>Mehl</li>
-        <li>Wasser</li>
-        <li>Salz</li>
-        <li>Hefe</li>
-        <li>Sack Kartoffeln</li>
-        <li>Butter</li>
+        {Ingredients?.map((ingredient) => (
+          <li key={id}>{ingredient}</li>
+        ))}
       </Ingredient>
     </>
   );
 };
 
+RecipeIngredients.propTypes = {
+  Ingredients: PropTypes.array,
+  id: PropTypes.number,
```

Comment:
If your components are ready you could consider to append `.isRequired` to your PropTypes

Explanation:
- https://blog.logrocket.com/validating-react-component-props-with-prop-types-ef14b29963fc/

Reply:
thanks :)